apps/create-fuels-counter-guide/playwright.config.ts

```diff
@@ -0,0 +1,30 @@
+import { defineConfig, devices } from '@playwright/test';
+
+/**
+ * See https://playwright.dev/docs/test-configuration.
```

Comment:
This link is broken

Reply:
https://playwright.dev/docs/test-configuration

Does this not work for you?

Reply:
I think it must've been pasted with the `.` at the end, probably best to remove that.

Reply:
I haven't created this file though, it's generated by playwright. It's the default config

Reply:
Also the link does work fine if you cmd + click it from vscode. I don't think we need to change anything